recipe/meta.yaml

```diff
@@ -12,7 +12,7 @@ source:
     - 0001_use-setuptools.patch
 
 build:
-  number: 0
+  number: 2
```

Comment:
Sorry, what I meant to say is I don't think we need a build number bump. Though if we did, it should increase by 1. Could you please expand on why you think a build number bump is necessary here?